infra/modules/providers/azure/data-factory/terraform.tfvars

```diff
@@ -0,0 +1,12 @@
+resource_group_name                       = ""
```

Comment:
What is the benefit of having this file?

Reply:
My idea is to have a sample on how the inputs should look like.

Reply:
This might actually default a lot of variables to `""`, but I'm not sure. Can you maybe rename it to `terraform.tfvars.template`?